dashboard/app/models/regional_partner_organizer.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: regional_partner_organizers
+#
+#  id                  :integer          not null, primary key
+#  organizer_id        :integer          not null
+#  regional_partner_id :integer          not null
+#
+# Indexes
+#
+#  index_regional_partner_organizers_on_organizer_id         (organizer_id)
+#  index_regional_partner_organizers_on_regional_partner_id  (regional_partner_id)
+#
+
+class RegionalPartnerOrganizer < ActiveRecord::Base
```

Comment:
same as above. I think we should rename. These users don't _organize_ the partner, they are associated with the partner as program managers, and the partner entity can organize a workshop.